.github/scripts/update_versions.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+import sys
+import pathlib
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+import semver
+
+from ruamel.yaml import YAML
+
+
+yaml = YAML(typ='rt')
+yaml.indent(mapping=2, sequence=4, offset=2)
+PREFIX = pathlib.PurePath('charts', 'netdata')
+
+
+def _parse_command_line():
+    """Defines arguments for the program and parses the given command line"""
+    parser = ArgumentParser(
+        formatter_class=RawDescriptionHelpFormatter,
+        description=("""\
+             Specify  arguments as listed below
+             Examples:
+             --------------------------------
+                python update_version.py get_chart_version
+                python update_version.py set_chart_version
+                python update_version.py get_app_version
+                python update_version.py set_app_version --version v1.31.0
+    """))
+
+    parser.add_argument('action',
+                        choices=['get_chart_version', 'set_chart_version',
+                                 'get_app_version', 'set_app_version'])
+    parser.add_argument(
+                '--version',
+                required=False,
+                type=str,
+                nargs='?',
+                help='Version to set.')
+    return parser.parse_args()
+
+
+class Versioning:
+    """ Getting current Chart and App versions and bumping to new ones """
+    def __init__(self, chart_data, action, version=None):
+        self.action = action
+        self.version = version
+        self.chart_data = chart_data
+
+    def run(self):
+        getattr(self, self.action)()
+
+    def get_chart_version(self):
+        print(self.chart_data['version'])
+
+    def set_chart_version(self):
+        old_chart_data = self.chart_data['version']
+        new_chart_version = semver.VersionInfo.parse(old_chart_data).next_version(part='patch')
+
+        FileHandler.write_readme(old_chart_data, new_chart_version)
+        self.chart_data['version'] = str(new_chart_version)
+        FileHandler.write_chart(self.chart_data)
+
+    def get_app_version(self):
+        print(self.chart_data['appVersion'])
+
+    def set_app_version(self):
+        new_app_version = self.version
+        FileHandler.write_readme(self.chart_data['appVersion'],
+                                 new_app_version)
+        self.chart_data['appVersion'] = new_app_version
```

Comment:
```suggestion
        self.chart_data['appVersion'] = 'v' + new_app_version
```

Just noticed there is `v` prefix

https://github.com/netdata/helmchart/blob/6dc7e8fcdfc8c58815f3de7332de9335cd721ac0/charts/netdata/Chart.yaml#L28

Reply:
Approving after the fix.

Reply:
Yes but both the current and new version will be supply with the `v` prefix . if you notice for `appVersion` i do string replace and not actual version bump, This value will come from [netdata repository agent bump  PR workflow](https://github.com/netdata/netdata/blob/master/.github/workflows/docker.yml#L130)

Reply:
I mean not readme, but Chart.yml

Reply:
```diff
0 helmchart (issue-1506/Automate_version_bumping %)$ .github/scripts/update_versions.py set_app_version --version 1.99.0
0 helmchart (issue-1506/Automate_version_bumping *%)$ git diff
diff --git a/charts/netdata/Chart.yaml b/charts/netdata/Chart.yaml
index fa9960d..2b3579d 100644
--- a/charts/netdata/Chart.yaml
+++ b/charts/netdata/Chart.yaml
@@ -25,4 +25,4 @@ maintainers:
   - name: Oleksii Kravchenko
     email: kamikaze.ua@gmail.com
 icon: https://netdata.github.io/helmchart/logo.png
-appVersion: v1.31.0
+appVersion: 1.99.0
diff --git a/charts/netdata/README.md b/charts/netdata/README.md
index 7380a29..6ef98a5 100644
--- a/charts/netdata/README.md
+++ b/charts/netdata/README.md
@@ -1,6 +1,6 @@
 # Netdata Helm chart for Kubernetes deployments

-[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/netdata)](https://artifacthub.io/packages/search?repo=netdata) ![Version: 3.7.1](https://img.shields.io/badge/Version-3.7.1-informational) ![AppVersion: v1.31.0](https://img.shields.io/badge/AppVersion-v1.31.0-informational)
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/netdata)](https://artifacthub.io/packages/search?repo=netdata) ![Version: 3.7.1](https://img.shields.io/badge/Version-3.7.1-informational) ![AppVersion: 1.99.0](https://img.shields.io/badge/AppVersion-1.99.0-informational)

 _Based on the work of varyumin (https://github.com/varyumin/netdata)_.
```

Reply:
Ok, it depends on `${{ github.event.inputs.agent_version }}`. If it has `v` feel free to close this converstion.

Reply:
@ilyam8 No worries thanks for the review